src/Driver/driver_configs.jl

```diff
@@ -135,13 +149,17 @@ function AtmosLESConfiguration(
     numerical_flux_gradient = CentralNumericalFluxGradient(),
 ) where {FT <: AbstractFloat}
 
-    (polyorder_horz, polyorder_vert) = isa(N, Int) ? (N, N) : N
+    (polyorder_horiz, polyorder_vert) = isa(N, Int) ? (N, N) : N
+    # Check if polynomial degree was passed as a CL option
+    if ClimateMachine.Settings.degree != (-1, -1)
+        (polyorder_horiz, polyorder_vert) = ClimateMachine.Settings.degree
+    end
```

Comment:
```suggestion
    (polyorder_horiz, polyorder_vert) = get_polyorders(N)
```
Where:
```julia
function get_polyorders(N)
    if ClimateMachine.Settings.degree != (-1, -1)
        ClimateMachine.Settings.degree
    elseif N isa Int
        (N, N)
    else
        N
    end
end
```
And avoid the repetitions below. But it's not a biggie.

Reply:
Thanks for the suggestion! I put it in [5194752](https://github.com/CliMA/ClimateMachine.jl/pull/1845/commits/519475263f3b5476e35fe225bbcd1c9aa853fbf0)